monitor_check.py

```diff
@@ -201,6 +208,55 @@ async def guess_reason(session, url, http_semaphore):
             }
     return None
 
+async def guess_missing_dependency(session, package, build, http_semaphore):
+    url = builderlive_link(package, build)
+    try:
+        content = await fetch(session, url, http_semaphore)
+    except aiohttp.client_exceptions.ClientPayloadError:
+        logger.debug('broken content %s', url)
+        return False
+    match = re.search(r"Problem: package (.*?) requires", content)
+    if match:
+        pkg = pkgname(match.group(1))
+        if pkg not in missing_dependencies:
+            missing_dependencies[pkg] = []
+            missing_dependencies[pkg].append(package)
+        else:
+            if package not in missing_dependencies[pkg]:
+                missing_dependencies[pkg].append(package)
+    else:
+        missing_dependencies['match_failed'].append(package)
+
+def print_dependency_tree():
+    root = Node("root")
```

Comment:
There is a package called root in our package set. Let's not confuse this with the root package?

```suggestion
    root = Node("/")
```